src/SFA.DAS.Commitments.Api.Types/DataLock/Types/DataLockErrorCode.cs

```diff
@@ -1,20 +1,32 @@
 ﻿using System;
+using System.ComponentModel;
 
 namespace SFA.DAS.Commitments.Api.Types.DataLock.Types
 {
+    // https://skillsfundingagency.atlassian.net/wiki/spaces/DAS/pages/104941512/Processing+ILR+submissions+and+data+locks
     [Flags]
     public enum DataLockErrorCode
     {
         None = 0,
+        //[Description("No matching record found in an employer digital account for the UKPRN")]
```

Comment:
if we are adding these descriptions, is it worth uncommenting them? as we expand the integration tests we may want visibility of what these actually mean in some kind of output.

Reply:
I was in two minds whether to include them or not, and I thought if they're not used, don't take up space in the assembly, but I don't mind including the attributes, so they're now included.